java/code/src/com/redhat/rhn/frontend/action/systems/sdc/SystemHardwareAction.java

```diff
@@ -191,8 +191,12 @@ private void setupForm(HttpServletRequest request, CPU cpu, Server server,
         Collections.sort(nicList);
 
         List nicList2 = new ArrayList();
+        List nicList3 = new ArrayList();
+        List nicList4 = new ArrayList();
         for (String nicName : nicList) {
             NetworkInterface n = server.getNetworkInterface(nicName);
+            Boolean hasIPv4 = false;
+            Boolean hasIPv6 = false;
```

Comment:
Nitpick: Use boolean (primitive type) instead of Boolean(wrapper), it's cheaper. Boolean will make sense if you would have been using some of its useful utility methods or there was the possibility of having null which is not the case here.